website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -628,6 +694,47 @@ the transform itself as an argument, and the operation returns the output
 [Output PCollection] = await [Input PCollection].applyAsync([AsyncTransform])
 {{< /highlight >}}
 
+{{< highlight yaml >}}
+pipeline:
+  transforms:
+    ...
+    - name: ProducingTransform
+      type: ProducingTransformType
+      ...
+
+    - name: MyTransform
+      type: MyTransformType
+      input: ProducingTransform.output_name
+      ...
+{{< /highlight >}}
+
+{{< paragraph class="language-yaml">}}
+The `.output_name` designation can be omitted for those transforms
+with a single (non-error) output.
+{{< /highlight >}}
```

Comment:
This seems a bit confusing - it reads like it is typical to specify the output, when in practice this is rarely done (except error handling). What if the logic was reversed by having the example be omitting the named output, and then mentioning that transforms with multiple outputs can use the `.output_name` notation (and link to error_handling doc)

Reply:
Good point. Done.